<launch>
    <arg name="task" default="navigation" />

    <!-- Launch the counter node -->
    <node pkg="evaluation_nodes" type="fre_counter_node" name="fre_counter" />

    <!-- Launch the detection spawner -->
    <node pkg="evaluation_nodes" type="detection_spawner.py" name="detection_spawner" if="$(eval arg('task') == 'mapping')" />

    <!-- Launch RQT -->
    <node pkg="rqt_gui" type="rqt_gui" name="dashboard" args="--standalone evaluation_nodes/Evaluation --args --rviz_config '$(find virtual_maize_field)/rviz/config.rviz' --task $(arg task)" output="screen" />
</launch>
